.github/workflows/docker_image_public_release.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     secrets: inherit # pass all secrets
 
   e2e_integration_test:
+    if: github.ref == 'refs/heads/develop' # TODO: re-enable in SDP-968
```

Comment:
We should support `main` as well, since releases come out of the main branch.

Reply:
This workflow only runs for develop and sdp-multitenant right ?

Reply:
I'll just blacklist sdp-multi tenant